shared/js/background/settings.es6.js

```diff
@@ -95,5 +95,8 @@ module.exports = {
     updateSetting: updateSetting,
     removeSetting: removeSetting,
     logSettings: logSettings,
-    ready: ready
+    ready: ready,
+    isReady () {
+        return isReady
```

Comment:
I think we can drop this given: https://app.asana.com/0/inbox/1199237043596759/1200071625925781/1200075771973335

Reply:
yep, done.